lms/templates/footer.html

```diff
@@ -47,12 +51,31 @@
         </p>
       </div>
 
+      ## Translators: '&copy;' is an HTML character code for the copyright symbol
       <p class="copyright">${_("&copy; 2014 {platform_name}, some rights reserved").format(platform_name=settings.PLATFORM_NAME)}</p>
         <nav class="nav-legal">
           <ul>
+	    %if marketing_link('HONOR') and marketing_link('HONOR') != '#':
+            <li class="nav-legal-01">
+	      <%
+		 tos_link = u"<a href='{}'>".format(marketing_link('TOS'))
+		 honor_link = u"<a href='{}'>".format(marketing_link('HONOR'))
+	      %>
+	      ${
+	        _("{tos_link_start}Terms of Service{tos_link_end} and {honor_link_start}Honor Code{honor_link_end}").format(
```

Comment:
@singingwolfboy how's this?

I am having trouble, however, figuring out how to verify if this logic is even working when there is no honor code link present. Noting it down so I remember to look into it tomorrow - I thought i verified the similar logic at L17 and 24 above previously.